ui/app/pages/swaps/swaps.util.test.js

```diff
@@ -1,54 +1,31 @@
-import { strict as assert } from 'assert';
-import proxyquire from 'proxyquire';
+import nock from 'nock';
 import { MAINNET_CHAIN_ID } from '../../../../shared/constants/network';
 import {
-  TRADES_BASE_PROD_URL,
-  TOKENS_BASE_PROD_URL,
-  AGGREGATOR_METADATA_BASE_PROD_URL,
-  TOP_ASSET_BASE_PROD_URL,
   TOKENS,
   EXPECTED_TOKENS_RESULT,
   MOCK_TRADE_RESPONSE_2,
   AGGREGATOR_METADATA,
   TOP_ASSETS,
 } from './swaps-util-test-constants';
 
-const swapsUtils = proxyquire('./swaps.util.js', {
-  '../../helpers/utils/fetch-with-cache': {
-    default: (url, fetchObject) => {
-      assert.strictEqual(fetchObject.method, 'GET');
-      if (url.match(TRADES_BASE_PROD_URL)) {
-        assert.strictEqual(
-          url,
-          'https://api.metaswap.codefi.network/trades?destinationToken=0xE41d2489571d322189246DaFA5ebDe1F4699F498&sourceToken=0x617b3f8050a0BD94b6b1da02B4384eE5B4DF13F4&sourceAmount=2000000000000000000000000000000000000&slippage=3&timeout=10000&walletAddress=0xmockAddress',
-        );
-        return Promise.resolve(MOCK_TRADE_RESPONSE_2);
-      }
-      if (url.match(TOKENS_BASE_PROD_URL)) {
-        assert.strictEqual(url, TOKENS_BASE_PROD_URL);
-        return Promise.resolve(TOKENS);
-      }
-      if (url.match(AGGREGATOR_METADATA_BASE_PROD_URL)) {
-        assert.strictEqual(url, AGGREGATOR_METADATA_BASE_PROD_URL);
-        return Promise.resolve(AGGREGATOR_METADATA);
-      }
-      if (url.match(TOP_ASSET_BASE_PROD_URL)) {
-        assert.strictEqual(url, TOP_ASSET_BASE_PROD_URL);
-        return Promise.resolve(TOP_ASSETS);
-      }
-      return Promise.resolve();
-    },
-  },
-});
-const {
+import {
   fetchTradesInfo,
   fetchTokens,
   fetchAggregatorMetadata,
   fetchTopAssets,
-} = swapsUtils;
+} from './swaps.util';
+
+jest.mock('../../../lib/storage-helpers.js', () => ({
+  getStorageItem: jest.fn(),
+  setStorageItem: jest.fn(),
+}));
 
-describe('Swaps Util', function () {
-  describe('fetchTradesInfo', function () {
+describe('Swaps Util', () => {
+  afterAll(() => {
+    nock.cleanAll();
```

Comment:
It would be better to call this after each test, rather than after all of them, to ensure they don't incidentally rely upon an interceptor setup for a different test. Similarly, the interceptors being setup in `beforeAll` hooks in various places can be moved to `beforeEach`.